posts/2024-12-10_why-gno/README.md

```diff
@@ -0,0 +1,102 @@
+  - Decentralization of the hosting and the governance, but centralization of the communication platform. Thanks to the nature of blockchains, we can create social platforms which are hosted by many different validators, all with the same exact state. The governance can be apart from the validators, and be guided by different principles and incentives.
+      - With the Fediverse, we share avoiding single points of failure and centralization of governance into a single entity (person or company).
+      - As opposed to the Fediverse, gno.land platforms can be more reliable and be centralized in terms of discovering and interacting with other users.
+  - Funding by the users. Align the platforms we use with the users' interests, rather than the advertisers'.
+      - This happens practically by ensuring the transaction fees, or revenue models build on top of them, can adequately fund both the hosting and the governance/moderation.
+  - Freedom of speech. Guarantee a baseline of moderation to remove outright harmful and illegal content, and hand off moderation to the choices of smaller communities, algorithms and moderating systems built on top.
+    - The chain should incentivize thoughtful content creation, moderation, and active participation in the community; but not block out content, so that the platform can be truly "neutral".
+      - There can and should be algorithms that show users interesting content while blocking out what they don't want to see. But users need to be able to choose between different algorithms and understand what each one is targeting; rather than solely signing up for the one that maximises "engagement" or "screen-time".
+- Why gno.land can achieve it:
+  - By solving the incentive problem, we ensure that the chain can continue to work with a specific vision while being hard to subvert. The governance aims to expand the blockchain's potential and usages in real applications, rather than the capital of its investors.
+  - By solving the technical problem, we can actually create a platform with the properties described above.
+  - gno.land pays attention to creating scalable data structures, that can host the content of the platform and enable free and universal access to it.
+
+## The Incentive Problem
+
+> Establishing Anti-Subversion Governance.
+
+Most blockchain incentive schemes focus primarily on rewarding validators for securing consensus and maintaining network integrity.  In order to build larger systems which are not just financially focused, we need to incentivize non-validators who nonetheless contribute to the advancement of the project, and make sure the chain governance is philosophically aligned, rather than just looking out for individual gains.
```

Comment:
```suggestion
Most blockchain incentive schemes focus primarily on rewarding validators for securing consensus and maintaining network integrity. In order to build larger systems which are not just financially focused, we need to incentivize non-validators who nonetheless contribute to the advancement of the project, and make sure the chain governance is philosophically aligned, rather than just looking out for individual gains.
```